src/engine/enginebuffer.cpp

```diff
@@ -1179,10 +1177,13 @@ void EngineBuffer::processSeek(bool paused) {
     }
 
     if (!paused && (seekType & SEEK_PHASE)) {
-        position = m_pBpmControl->getNearestPositionInPhase(position, true, true);
+        double syncPosition = position;
```

Comment:
I think this assignment is pointless. Can we initialize syncPosition two lines below?